### ISSUE TITLE
bobbyrenwick.pip is fully broken:  The error was: error while evaluating conditional (pip_latest_output.stdout.find('Requirement already up-to-date') == -1): 'dict object' has no attribute 'stdout'"}

### ISSUE THREAD

**Issue by ssbarnea** [bobbyrenwick/ansible-pip#13]
This happens on OS X and Ubuntu too, I tried to install this role using this:

```
dependencies:
    - role: bobbyrenwick.pip
      python: python
      pip: pip
    - role: bobbyrenwick.pip
      python: python3
      pip: pip3
```

Errors:

```
00:00:02.466 TASK [bobbyrenwick.pip : Upgrade to latest version of pip.] ********************
00:00:02.523 fatal: [localhost]: FAILED! => {"failed": true, "msg": "The conditional check 'pip_latest_output.stdout.find('Requirement already up-to-date') == -1' failed. The error was: error while evaluating conditional (pip_latest_output.stdout.find('Requirement already up-to-date') == -1): 'dict object' has no attribute 'stdout'"}
00:00:02.524 
```

The code of the failing task:

```
- name: Upgrade to latest version of pip.
  command: "{{ pip }} install -U pip"
  register: pip_latest_output
  become: yes
  changed_when: pip_latest_output.stdout.find('Requirement already up-to-date') == -1
  when: pip_version == None or pip_version == "LATEST"
```

**Comment by ssbarnea** [comment]
In fact I am not surprised, as you are registering the only in some cases, check the `when:`

**Comment by nmusatti** [comment]
In case you're still interested: I believe the first invocation in your example would fail the above error message on an installation with "pip3" but no "pip" executable. Could this be your case?

**Comment by nmusatti** [comment]
Should be fixed by #33 . If this isn't the case, let us know.
